app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -87,7 +89,12 @@ class TabTrayView(
     private var tabsTouchHelper: TabsTouchHelper
     private val collectionsButtonAdapter = SaveToCollectionsButtonAdapter(interactor, isPrivate)
 
-    private val syncedTabsController = SyncedTabsController(lifecycleOwner, view, store, concatAdapter)
+    private val syncedTabsController = SyncedTabsController(
```

Comment:
Is this change needed? Seems like only the formatting was changed.
I'd prefer sticking as much as possible to the original code to preserve commit history and so to have an easier life later when wearing the code archaeologist hat.